setup.py

```diff
@@ -0,0 +1,20 @@
+import setuptools
+
+setuptools.setup(
+    name="simGL", 
+    version="0.1",
+    author="Moisès Coll Macià, Graham Gower",
+    author_email="moicoll@birc.au.dk",
+    description="Simulate genotype likelihoods from haplotype genotype matrix.",
+    packages=["simGL"],
+    license = "ISC",
+    url = "https://github.com/RacimoLab/simGL",
+    install_requires=["numpy>=1.22","scipy>=1.7"],
+    classifiers = [
+        "Development Status :: 2 - Pre-Alpha",
+        "Operating System :: OS Independent",
+        "License :: OSI Approved :: ISC License",
+        "Intended Audience :: Science/Research",
+        "Programming Language :: Python :: 3.9",
```

Comment:
Do you want to support other python versions? Usually they're listed here too. And then you'll want to add a `python_requires = >=3.something` line, like e.g. in the demesdraw setup.cfg.